isPerfectSquare/isPerfectSquare.js

```diff
@@ -0,0 +1,51 @@
+/*
+Given a positive integer num, write a function which returns True if num is a perfect square else False.
+
+Follow up: Do not use any built-in library function such as sqrt.
+
+ 
+
+Example 1:
+
+Input: num = 16
+Output: true
+Example 2:
+
+Input: num = 14
+Output: false
+ 
+
+Constraints:
+
+1 <= num <= 2^31 - 1
+*/
+
+const isPerfectSquare = function(num) {
+  let low = 0;
```

Comment:
Why do you need to do recursion and all this couldn't you just return this  Math.sqrt(n) % 1 === 0;